fl4health/clients/moon_client.py

```diff
@@ -25,33 +22,58 @@ def __init__(
         loss_meter_type: LossMeterType = LossMeterType.AVERAGE,
         checkpointer: Optional[ClientCheckpointModule] = None,
         temperature: float = 0.5,
-        contrastive_weight: Optional[float] = None,
+        contrastive_weight: float = 1.0,
         len_old_models_buffer: int = 1,
     ) -> None:
+        """
+        This client implements the MOON algorithm from Model-Contrastive Federated Learning. The key idea of MOON
+        is to utilize the similarity between model representations to correct the local training of individual parties,
```

Comment:
nit: The key idea of MOON is to enforce similarity between representations from the global and local model through a contrastive loss to correct the local training of individual parties. Or something along those lines just to make it clear what the different models mean in this context

Reply:
👍